gpii/node_modules/solutionsRegistry/src/schemas/solution-schema.json5

```diff
@@ -49,7 +49,14 @@
                         "type": "object",
                         "additionalProperties": {
                             "type": "object",
-                            "additionalProperties": false,
+                            "additionalProperties": {
+                                "properties": {
+                                    "label": {
+                                        "type": "string",
+                                        "required": false
+                                    }
+                                }
+                            },
                             "properties": {
                                  "schema": { "$ref": "gss-v7-full#", "required": true }
```

Comment:
The optional `label` element should be described above this line.

Reply:
Added label above schema there.